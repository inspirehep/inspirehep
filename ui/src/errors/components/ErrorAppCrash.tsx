import React, { Component } from 'react';
import { Row } from 'antd';

import error500Image from '../images/500.svg';
import ErrorPage from './ErrorPage';

class ErrorAppCrash extends Component {
  render() {
    return (
      <Row type="flex" justify="center">
        <ErrorPage
          message="Something went wrong"
          detail={
            <span>
              Please try again later, you can <a href="/">go back home</a> now.
            </span>
          }
          imageSrc={error500Image}
        />
      </Row>
    );
  }
}

export default ErrorAppCrash;
