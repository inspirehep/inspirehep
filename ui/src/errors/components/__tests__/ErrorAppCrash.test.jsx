import React from 'react';
import { render } from '@testing-library/react';
import ErrorAppCrash from '../ErrorAppCrash';

describe('ErrorAppCrash', () => {
  it('renders ErrorAppCrash', () => {
    const { asFragment } = render(<ErrorAppCrash />);
    expect(asFragment()).toMatchSnapshot();
  });
});
