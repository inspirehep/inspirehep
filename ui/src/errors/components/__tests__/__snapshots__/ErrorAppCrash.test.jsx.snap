// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorAppCrash renders ErrorAppCrash 1`] = `
<Row
  justify="center"
  type="flex"
>
  <ErrorPage
    detail={
      <span>
        Please try again later, you can 
        <a
          href="/"
        >
          go back home
        </a>
         now.
      </span>
    }
    imageSrc="500.svg"
    message="Something went wrong"
  />
</Row>
`;
